mmv1/templates/terraform/examples/gkeonprem_vmware_cluster_basic.tf.erb

```diff
@@ -0,0 +1,77 @@
+resource "google_gkeonprem_vmware_cluster" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
For the basic example can we specify as little as possible- basically, just satisfy any required or exactly_one_of constraints?

If keeping this specific example around is valuable, we can rename it.